src/pages/leejiyon/Login/Login.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './login.scss';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  onChangeInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
```

Comment:
구조분해 할당 (비구조화 할당)을 사용해볼 수 있을 것 같아요 ! 근데 짧아서 굳이 싶기도 하고 🤔
```suggestion
  onChangeInput = e => {
    this.setState({
      const { name, value } = e.target;
      [name]: value,
    });
  };
```

Reply:
확인했습니다 !